GTG/gtk/backends_dialog.ui

```diff
@@ -51,59 +53,49 @@
                       </packing>
                     </child>
                     <child>
-                      <object class="GtkAlignment" id="alignment2">
-                        <property name="height_request">30</property>
+                      <object class="GtkToolbar" id="service_toolbar">
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
-                        <property name="yalign">1</property>
-                        <property name="top_padding">20</property>
-                        <property name="bottom_padding">10</property>
-                        <property name="left_padding">10</property>
-                        <property name="right_padding">10</property>
+                        <property name="toolbar_style">icons</property>
+                        <property name="show_arrow">False</property>
+                        <property name="icon_size">1</property>
                         <child>
-                          <object class="GtkButtonBox" id="hbuttonbox3">
+                          <object class="GtkToolButton" id="add_button">
                             <property name="visible">True</property>
                             <property name="can_focus">False</property>
-                            <property name="spacing">10</property>
+                            <property name="is_important">True</property>
```

Comment:
Sorry for late notice of this, this property is deprecated though, too:
https://developer.gnome.org/gtk3/stable/GtkAction.html#GtkAction--is-important